apis/apiextensions/v1/composition_types_test.go

```diff
@@ -489,6 +503,127 @@ func TestConvertResolve(t *testing.T) {
 	}
 }
 
+func TestGetConstantValue(t *testing.T) {
+	type args struct {
+		ct ConstantValue
+	}
```

Comment:
Nit: Usually when there's only one arg we eschew the `args` struct, so in this case we'd have ` ct` field directly in the `map[string]struct{}` of cases. That said, there's no harm in keeping it.